pages/common/vela.md

```diff
@@ -0,0 +1,8 @@
+# vela
+
+> Command line tools for Vela pipeline.
+> More information: <https://go-vela.github.io/docs/>.
+
+- Trigger pipeline to run from git tagged version:
+
+`vela add deployment --org {{organization}} --repo {{repository_name}} --target {{dev/stage/prod/other}} --ref refs/tags/{{git_tag}} --description "{{deploy_description}}"`
```

Comment:
Can you add more examples? Maybe 1 for github org, another for prod environment, Can I deploy using a branch?
```suggestion
`vela add deployment --org {{organization}} --repo {{repository_name}} --target {{environment}} --ref refs/tags/{{git_tag}} --description "{{deploy_description}}"`
```
Also Vela has more subcommands like `view` and `get`

Reply:
I can add examples for `view` and `get` (although I'll just copy from the docs page since I personally have not used those commands before).  I did just check, it is commit/branch/ or tag for the `--ref` option